runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -185,6 +186,26 @@ public void mouseReleased(MouseEvent e)
 		add(container, BorderLayout.CENTER);
 	}
 
+	private static BufferedImage createHoverImage(BufferedImage image)
```

Comment:
Nothing else you changed uses this function, and I don't see a difference. Why did you move this?

Reply:
I believe it was due to code styling preferences automatically hoisting static methods to the top of the file. I left it in because I thought it made sense. I could revert the change.

Reply:
That explains all the sudden whitespace removals in the xp tracker plugin around the private variables.